<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>131</red>
        <green>173</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>131</red>
        <green>173</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>131</red>
        <green>173</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>131</red>
        <green>173</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="artirBtn">
   <property name="geometry">
    <rect>
     <x>832</x>
     <y>490</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bir Sonraki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>256</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1- Oda Işığını Aç</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2- Oda Işığını Kapat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3- Gece Lambasını Aç</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4- Gece Lambasını Kapat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5- Perdeyi Aç</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6- Perdeyi Kapat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7- Televizyon Aç</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8- Kitap Oku</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9- Haberlere Gözat</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="azaltBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bir Önceki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nightBulbLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>300</width>
     <height>213</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="curtainLbl">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>300</y>
     <width>170</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tvLbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onaylaBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ONAYLA</string>
   </property>
  </widget>
  <widget class="QLabel" name="frameLbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="uykuModuBtn">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>850</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>UYKU MODU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
